test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperBlockRewriterTest/TagHelperParseTreeRewriter_RewritesComplexAttributeTagHelperTagBlocks1.stree.txt

```diff
@@ -1,23 +1,61 @@
-Markup block - Gen<None> - 51 - (0:0,0)
-    Tag block - Gen<TagHelper> - 51 - (0:0,0) - p - ptaghelper
-        StartTagAndEndTag - <p class="@DateTime.Now" style='@DateTime.Now'> ... </p>
-        class - DoubleQuotes
-            Markup block - Gen<None> - 13 - (10:0,10)
-                Markup block - Gen<DynAttr:@(10:0,10)> - 13 - (10:0,10)
-                    Expression block - Gen<Expr> - 13 - (10:0,10)
-                        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (10:0,10) - Tokens:1
-                            SyntaxKind.Transition;[@];
-                        Code span - Gen<Expr> - [DateTime.Now] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (11:0,11) - Tokens:3
-                            SyntaxKind.Identifier;[DateTime];
-                            SyntaxKind.Dot;[.];
-                            SyntaxKind.Identifier;[Now];
-        style - SingleQuotes
-            Markup block - Gen<None> - 13 - (32:0,32)
-                Markup block - Gen<DynAttr:@(32:0,32)> - 13 - (32:0,32)
-                    Expression block - Gen<Expr> - 13 - (32:0,32)
-                        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (32:0,32) - Tokens:1
-                            SyntaxKind.Transition;[@];
-                        Code span - Gen<Expr> - [DateTime.Now] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (33:0,33) - Tokens:3
-                            SyntaxKind.Identifier;[DateTime];
-                            SyntaxKind.Dot;[.];
-                            SyntaxKind.Identifier;[Now];
+RazorDocument - [0..51)::51 - [<p class="@DateTime.Now" style='@DateTime.Now'></p>]
+    MarkupBlock - [0..51)::51
+        MarkupTagHelperElement - [0..51)::51 - p[StartTagAndEndTag] - ptaghelper
+            MarkupTagHelperStartTag - [0..47)::47
+                MarkupTextLiteral - [0..2)::2 - [<p] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                    OpenAngle;[<];
+                    Text;[p];
+                MarkupTagHelperAttribute - [2..24)::22 - class - DoubleQuotes - [ class="@DateTime.Now"]
```

Comment:
It'd be valuable in the baselines to understand which attributes are bound and which ones aren't.

Reply:
Good idea. Doing this now